strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -53,8 +65,26 @@ public void configureMessageConverters(List<HttpMessageConverter<?>> converters)
         converters.add(new ByteArrayHttpMessageConverter()); // if your argument is a byte[]
         converters.add(stringConverter);
         converters.add(new FormHttpMessageConverter());
-        converters.add(new MappingJackson2HttpMessageConverter());
+        converters.add(jackson2Converter());
         converters.add(jaxb2RootElementHttpMessageConverter);
         converters.add(new ResourceHttpMessageConverter());
     }
+
+    @Bean
+    public MappingJackson2HttpMessageConverter jackson2Converter()
+    {
+        MappingJackson2HttpMessageConverter converter = new MappingJackson2HttpMessageConverter();
+        converter.setObjectMapper(objectMapper);
+        return converter;
+    }
+
+    @Override
+    public void configurePathMatch(PathMatchConfigurer configurer)
+    {
+        configurer
+                .setUseSuffixPatternMatch(true)
+                .setUseTrailingSlashMatch(false)
+                .setUseRegisteredSuffixPatternMatch(true)
+                .setPathMatcher(antPathMatcher);
+    }
```

Comment:
Cosmetic/Code style:

For all similar places, could you please align the methods based on the first one's starting index like this?

       configurer.setUseSuffixPatternMatch(true)
                 .setUseTrailingSlashMatch(false)
                 .setUseRegisteredSuffixPatternMatch(true)
                 .setPathMatcher(antPathMatcher);